compact/pid/pfrich.xml

```diff
@@ -88,6 +88,13 @@
 <constant name="AEROGEL_SEPARATOR_WALL_THICKNESS" value="0.05"/>
 <constant name="AEROGEL_OUTER_WALL_THICKNESS" value="0.1"/>
 
+<!-- HRPPD material definitions -->
+
+<constant name="PFRICH_HRPPD_WindowMat" value="Quartz"/>
+<constant name="PFRICH_HRPPD_PCBMat" value="G10"/>
+<constant name="PFRICH_HRPPD_MPDMat" value="SiliconDioxide"/>
+<constant name="PFRICH_HRPPD_ASICMat" value="SiliconCarbide"/>
```

Comment:
These values should not come from constants in `<define>`, but defined in xml inside the `<detector>` definition below.